lib/slack.ex

```diff
@@ -1,6 +1,6 @@
 defmodule Slack do
   @moduledoc """
-  A client for the Slack API.
+  A client for the Slack API.
```

Comment:
Is this a whitespace change of some kind?

Reply:
Yep - it was a while back but if I recall it was because the latest compiler complains about mis-indented heredoc strings.